From be313683c6cd7a8895e61d729f9429db8f517440 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Wed, 16 Mar 2022 16:10:17 +0530
Subject: [PATCH] Update clang library version

Signed-off-by: svenate <salini.venate@intel.com>
---
 core/definitions.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/definitions.mk b/core/definitions.mk
index 5a4a9f8..568407e 100755
--- a/core/definitions.mk
+++ b/core/definitions.mk
@@ -88,9 +88,9 @@ IAFW_CC := $(IAFW_TOOLS_CLANG_PREFIX)/clang
 IAFW_OBJCOPY := $(IAFW_TOOLS_GCC_PREFIX)objcopy$(HOST_EXECUTABLE_SUFFIX)
 EFI_OBJCOPY := $(IAFW_OBJCOPY)
 ifeq ($(TARGET_IAFW_ARCH),x86_64)
-IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib64/clang/12.0.7/lib/linux/libclang_rt.builtins-x86_64-android.a
+IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib64/clang/14.0.2/lib/linux/libclang_rt.builtins-x86_64-android.a
 else
-IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib64/clang/12.0.7/lib/linux/libclang_rt.builtins-i686-android.a
+IAFW_LIBCLANG := $(IAFW_TOOLCHAIN_CLANG_ROOT)/../lib64/clang/14.0.2/lib/linux/libclang_rt.builtins-i686-android.a
 endif
 
 # Transformation definitions, ala build system's definitions.mk
-- 
2.17.1

