From ba80c67d5d2fbacc82b7d46176fc24331121fb1a Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Wed, 16 Mar 2022 00:48:44 +0530
Subject: [PATCH] Update clang version to latest

Signed-off-by: svenate <salini.venate@intel.com>
---
 groups/kernel/AndroidBoard.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index 0adc123..d3aaa57 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -22,7 +22,7 @@ kernel: $(PRODUCT_OUT)/kernel
 
 else
 
-TARGET_KERNEL_CLANG_VERSION := r416183b1
+TARGET_KERNEL_CLANG_VERSION := r445002
 CLANG_PREBUILTS_PATH := $(abspath $(INTEL_PATH_DEVICE)/../../../prebuilts/clang)
 
 ifneq ($(TARGET_KERNEL_CLANG_VERSION),)
-- 
2.17.1

