From 3439abe8b6826e50f03208cdde6aa82ebfcfc056 Mon Sep 17 00:00:00 2001
From: ahs <amrita.h.s@intel.com>
Date: Tue, 17 Mar 2020 17:43:04 +0530
Subject: Link with libkmod-shared

Tracked-On:
Signed-off-by: ahs <amrita.h.s@intel.com>
---
 Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 344979d..93a94bb 100644
--- a/Android.bp
+++ b/Android.bp
@@ -14,7 +14,7 @@ cc_binary {
     ],
 
     shared_libs: [
-        "libkmod",
+        "libkmod-shared",
         "libcutils",
     ],
 }
-- 
2.24.0

